markdown/adresse_pflichtfelder.md

```diff
@@ -0,0 +1,17 @@
+|**Datum**|**Autor**|**bestehende Doku**|**Link**|
+|---------|---------|-------------------|--------|
+|09. Dezember 2020|Hochfrequenz Unternehmensberatung GmbH|[bo4e.de](https://www.bo4e.de/dokumentation/komponenten/com-adresse)|[Github](https://github.com/Hochfrequenz/bo4e-modification-proposals/blob/master/markdown/adresse_pflichtfelder.md)|
+
+# Anpassung der Pflichtfelder im COM Adresse
+Wir schlagen vor, in der Komponente `Adresse` die Strasse und Hausnummer nicht mehr als Pflichtfelder anzugeben.
```

Comment:
```suggestion
Wir schlagen vor, in der Komponente `Adresse` die Strasse und Hausnummer nicht mehr als Pflichtfelder anzugeben.
Wir schlagen außerdem vor analog zur `messadresse`/`geoadresse`/`katasteradresse` im BO Messlokation eine XOR-Validierung einzuführen, dass entweder Straße und Hausnummer (exklusiv) oder ein Postfach angegeben werden muss.
```

Reply:
Kennt sich jemand von uns gut mit UML Diagrammen aus?
Würde das gerne auch entsprechend mal in ein UML Diagramm gießen, dass eins der genannten Felder pflicht ist und zwar genau eins.

Reply:
```suggestion
Wir schlagen vor, in der Komponente `Adresse` die Strasse und Hausnummer nicht mehr als Pflichtfelder anzugeben.
Wir schlagen außerdem vor analog zur `messadresse`/`geoadresse`/`katasteradresse` im BO Messlokation eine XOR-Validierung einzuführen, dass entweder Straße und Hausnummer (exklusiv) oder ein Postfach angegeben wird, wenn diese angegeben werden.
```
Man kann ja auch weder noch angeben.